documentation/develop/choosing-a-firefox-version-for-extension-development.md

```diff
@@ -102,19 +102,19 @@ Suggested development use: You may want to install and test a signed copy of you
+You load extensions under development into this version of Firefox using [about:debugging](https://developer.mozilla.org/docs/Tools/about:debugging) or [web-ext](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Getting_started_with_web-ext) and have access to the [addon debugging window](https://developer.mozilla.org/docs/Tools/Browser_Toolbox). You can also install unsigned extensions, after adjusting or setting certain about:config properties (generally fewer changes than you need to achieve the same tasks in Nightly). This enables you to test features such as restart behavior, permission prompts, and upgrades.
 
 Development of extensions for Firefox for Android is fully supported.
 
 Suggested development use: Use Developer Edition as your primary web extension development and testing platform.
 
-[Download](https://www.mozilla.org/en-US/firefox/channel/desktop/#developer)
+[Download](https://www.mozilla.org/firefox/channel/desktop/#developer)
 
 ### Nightly
 
 This version of Firefox provides a build that is updated nightly with the latest in development features for the future release of Firefox. It is generally used by people interested in experiencing the very latest features and providing feedback when they find issues.
 
-You load extensions under development into this version of Firefox using [about:debugging](https://developer.mozilla.org/en-US/docs/Tools/about:debugging) or [web-ext](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Getting_started_with_web-ext) and have access to the [addon debugging window](https://developer.mozilla.org/en-US/docs/Tools/Browser_Toolbox). You can also install unsigned extensions, after adjusting or setting certain about:config properties. This enables you to test features such as restart behavior, permission prompts, and upgrades.
+You load extensions under development into this version of Firefox using [about:debugging](https://developer.mozilla.org/docs/Tools/about:debugging) or [web-ext](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Getting_started_with_web-ext) and have access to the [addon debugging window](https://developer.mozilla.org/docs/Tools/Browser_Toolbox). You can also install unsigned extensions, after adjusting or setting certain about:config properties. This enables you to test features such as restart behavior, permission prompts, and upgrades.
```

Comment:
Please change web-ext link to `/documentation/develop/getting-started-with-web-ext/`.